CONTRIBUTING.md

```diff
@@ -0,0 +1,255 @@
+- [Introduction](#introduction)
+- [The contribution process](#the-contribution-process)
+  - [Raise an issue](#raise-an-issue)
+  - [Create a fork](#create-a-fork)
+  - [Create a new folder](#create-a-new-folder)
+  - [Add license](#add-license)
+  - [Create a notebook](#create-a-notebook)
+  - [Commit new changes](#commit-new-changes)
+  - [Open a pull request](#open-a-pull-request)
+  - [Common recommendations for the review](#common-recommendations-for-the-review)
+- [CI/CD test passing guide](#cicd-test-passing-guide)
+  - [PEP 8 Style](#pep-8-style)
+  - [Notebook execution](#notebook-execution)
+  - [Format requirements](#format-requirements)
+- [Benchmarking result report](#benchmarking-result-report)
+
+## Introduction
+
+Thanks for considering a contribution to the MONAI Tutorials Repository and reading the Contributing Guidelines.
+
+The following is a set of guidelines for contributing tutorials to Project MONAI.
+The guidelines are based on the discussions in #1119
+Please feel free to propose changes to this document in a pull request(PR).
+
+## The contribution process
+
+### Raise an issue
+
+We encourage you to [raise an issue](https://github.com/Project-MONAI/tutorials/issues/new/choose) about your needs for MONAI tutorials.
+The idea may generate echo in the community and interest other contributors as well.
+
+### Create a fork
+
+When you are ready to kick off some coding, [create a new fork](https://github.com/Project-MONAI/tutorials/fork) of the `main` branch of this repository.
+
+After forking is complete, you can `git clone` the forked repo to your development environment, and use `git checkout -b` to create a new branch to freely experiment with your changes.
+
+For example:
+```bash
+git clone https://github.com/<your github space>/tutorials.git
+git checkout -b <your new branch name>
+```
+
+### Create a new folder
+
+MONAI tutorials covered examples of various medical applications and technical topics.
+Tutorials under the same topic/application share the same folder under the root directory, e.g. `2d_classification`, `2d_segmentation`, `3d_classification`, and `3d_segmentation`.
+
+To hold your work in a new location, you can choose to create a new subfolder under the existing ones, or a folder under the root directory if it doesn't belong to any of the existing topics/applications.
+
+The folder should have a `README.md` file with descriptive information for others to start using your code or tutorial notebooks.
+
+Finally, the MONAI tutorial has a [README file](README.md) to communicate the important information and provide an overview of the tutorials and examples. For new tutorials, please add a new entry to the [List](README.md#4-list-of-notebooks-and-examples) of notebooks and examples](README.md#4-list-of-notebooks-and-examples).
```

Comment:
Missed "[" in "of notebooks and examples]".

Thanks.

Reply:
Fixed. Thanks!